databases/week2/ErDiagram.sql

```diff
@@ -0,0 +1,91 @@
+
+
+SET NAMES utf8mb4;
+
+CREATE TABLE `User` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `email` VARCHAR(255) NOT NULL,
+  `password` VARCHAR(32) NOT NULL,
+  `create_time` DATETIME NOT NULL,
+  `name` VARCHAR(45) NOT NULL,
+  PRIMARY KEY (`id`))
+ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `Occassion` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(255) NOT NULL,
+  PRIMARY KEY (`id`))
+ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+
+CREATE TABLE `Payment` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `amount` FLOAT NOT NULL,
+  `user_id` INT NOT NULL,
+  `Order_id` INT NOT NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `fk_user`
+    FOREIGN KEY (`user_id`)
+    REFERENCES `User` (`id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION,
+  CONSTRAINT `fk_Payment_Order`
```

Comment:
Complains that it can't create this foreign key. Do yo know why?

Reply:
ya the error is because Order table does not exist yet!. I have fixed this.

Reply:
:D